.github/RELEASE.md

```diff
@@ -0,0 +1,177 @@
+The complete LibreTime documentation is available at libretime.org.
+
+The full tarball for the 3.0.0-alpha.8 release of LibreTime is available here.
+
+Since this is an alpha release there will be bugs in the code.
+
+Please report new issues and/or feature requests in the issue tracker. Join our discourse or chat us up on Slack if you need help and for general discussion.
+
+## Table of Contents
+
+    Features
```

Comment:
Should all of these be internal links or should we remove the ToC?
```suggestion
    [Features](#Features)
```
(I didn't test linking as in the suggestion)

Reply:
This was a copy/paste of our previous release notes document; links probably won't be needed if we edit the content down to something like 300 words max.